tests/ui/missing-function/rust-by-example-description/main.rs

```diff
@@ -1,13 +0,0 @@
-// Copyright Kani Contributors
-// SPDX-License-Identifier: Apache-2.0 OR MIT
-
-// kani-flags: --enable-unstable --cbmc-args --unwind 4 --object-bits 9
-// This test is to check if the description for undefined functions has been updated to "Function with missing definition is unreachable"
```

Comment:
While this one doesn't work, shouldn't we keep a test for the property description? IIRC we had difficulty detecting it in the past. Or did this get replaced with a different test and I didn't notice

Reply:
I did keep two test for that. There is one test for the case of an `extern` function and one for the `--legacy-linker`

Reply:
- https://github.com/model-checking/kani/tree/main/tests/ui/missing-function/replaced-description
- https://github.com/model-checking/kani/tree/main/tests/ui/missing-function/extern_c